projects/web-app/src/views/apple-ios.view.vue

```diff
@@ -1,92 +1,95 @@
 <template>
-  <panel-section title="Properties">
-    <div v-if="!appleMetadata.length" class="warning-message">
-      <WarningIcon class="icon" />
-      <p>No meta properties detected.</p>
-    </div>
-    <properties-list v-else>
-      <properties-item
-        v-for="item in appleMetadata"
-        :key="item.keyName"
-        :key-name="item.keyName"
-        :refresh-on="appleMetadata"
-      >
-        <template #default>
-          {{ item.title }}
-        </template>
-        <template #value>
-          {{ item.value }}
-        </template>
-      </properties-item>
-    </properties-list>
-  </panel-section>
-  <panel-section title="Touch icons">
-    <div v-if="!touchIcons.length" class="warning-message">
-      <WarningIcon class="icon" />
-      <p>No touch icons detected.</p>
-    </div>
-    <properties-list v-else>
-      <properties-item
-        v-for="(icon, index) in touchIcons"
-        :key="index"
-        :key-name="icon.sizes"
-        :refresh-on="touchIcons"
-      >
-        <template #default>
-          <template v-if="icon.sizes">
-            {{ icon.sizes }}
+  <div>
+    <h1>Apple iOS</h1>
```

Comment:
Are you sure this needs to be an `<h1>`?

Reply:
Yep, dit is de enige H1 van de website, het wordt ook gebruikt als de page title.